src/connection/ClientConnection.cpp

```diff
@@ -98,13 +119,11 @@ bool ClientConnection::initializeRequest(int clientFD)
 	time_t  currentTime;
 	time(&currentTime);
 	size_t  headerEnd = client.receiveStr.find(CRLFCRLF);
-	size_t  sizeCRLFCRLF = strlen(CRLFCRLF);
 
 	if (headerEnd != std::string::npos) {
-		client.request = std::make_shared<Request>(client.receiveStr.substr(0, headerEnd + sizeCRLFCRLF), _log);
-		if (headerEnd + sizeCRLFCRLF < client.receiveStr.length()) {
-			client.request->appendToBody(client.receiveStr.substr(headerEnd + sizeCRLFCRLF));
-		}
+		client.request = std::make_shared<Request>(client.receiveStr.substr(0, headerEnd + CRLFCRLFsize), _log);
+		if (headerEnd + CRLFCRLFsize < client.receiveStr.length())
```

Comment:
CRLFCRLFsize undefined?

Reply:
CRLFCRLFsize zit nu in de defines.hpp. Lijkt mij niet nodig om daar elke keer de length van te berekenen.